index.html

```diff
@@ -15,12 +15,14 @@
     <script src="node_modules/chart.js/dist/Chart.js"></script>
     <script src="node_modules/urlhash/urlHash.js"></script>
     <script src="dist/community-toolbox.js" charset="utf-8"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.4.2/js/swiper.min.js"></script>
 
   </head>
 
   <body>
 
     <link href="node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.4.2/css/swiper.min.css">
```

Comment:
What if a developer is trying to contribute but inet is not working? The lib will also not be working and might even break the page by showing an error.

Reply:
ok 
what we can do is we will add these file node module but initially call from cdn directly
if they fail then only they will use node modules file ( download files only when fails not everytime)

Reply:
using cdn is good because files get cached and some library or framwork are used almost every website like bootstrap , font awesome so they are cached and our website loads fast
@HarshKhandeparkar

Reply:
Sending a request to another website is not good, multiple requests to the same site are not good anyways but cross site requests add even more latency, isn't there a way to cache local libs?

Reply:
@HarshKhandeparkar  
We can cache them in browser cache or storage ( ? not sure )

Reply:
> Sending a request to another website is not good, multiple requests to the same site are not good anyways but cross site requests add even more latency,
>

don't consider cdn server and webite servers same

Reply:
> > Sending a request to another website is not good, multiple requests to the same site are not good anyways but cross site requests add even more latency,
> 
> don't consider cdn server and webite servers same

in context of usage and multiple requests

Reply:
@HarshKhandeparkar   https://www.quora.com/Is-it-better-to-use-Bootstrap-offline-or-just-include-a-link-to-the-CDN